src/page/landing/LandingPage.tsx

```
@@ -14,6 +14,7 @@ import Step from "@/components/step";
 import Features from "@/components/ui/features";
 import Pricing from "@/components/ui/pricing";
 import Review from "@/components/review";
+import { Popover, PopoverContent, PopoverTrigger } from "@/components/ui/popover";
```

ESLint

[error] 'Popover' is defined but never used.

(@typescript-eslint/no-unused-vars)